packages/cli/src/commands/capabilities/translations.ts

```diff
@@ -0,0 +1,170 @@
+		'$ smartthings capabilities:translations 1 en',
+		'$ smartthings capabilities:translations custom1.outputModulation 1 1',
+		'$ smartthings capabilities:translations custom1.outputModulation 1 en',
+		'Tag: en',
+		'',
+		'Attributes:',
+		'┌────────────────────────┬───────────────────┬────────────────────────────────┬────────────────────────────────────────────────────┐',
+		'│ Name                   │ Label             │ Description                    │ Template                                           │',
+		'├────────────────────────┼───────────────────┼────────────────────────────────┼────────────────────────────────────────────────────┤',
+		'│ outputModulation       │ Output Modulation │ Power supply output modulation │ The {{attribute}} of {{device.label}} is {{value}} │',
+		'│ outputModulation.50hz  │ 50 Hz             │                                │                                                    │',
+		'│ outputModulation.60hz  │ 60 Hz             │                                │                                                    │',
+		'└────────────────────────┴───────────────────┴────────────────────────────────┴────────────────────────────────────────────────────┘',
+		'',
+		'Commands:',
+		'┌──────────────────────────────────────┬───────────────────────┬──────────────────────────────────────────────────┐',
+		'│ Name                                 │ Label                 │ Description                                      │',
+		'├──────────────────────────────────────┼───────────────────────┼──────────────────────────────────────────────────┤',
+		'│ setOutputModulation                  │ Set Output Modulation │ Set the output modulation to the specified value │',
+		'│ setOutputModulation.outputModulation │ Output Modulation     │ The desired output modulation                    │',
+		'└──────────────────────────────────────┴───────────────────────┴──────────────────────────────────────────────────┘',
+	]
+
+	primaryKeyName = 'id'
+	primaryKeyName2 = 'tag'
+	sortKeyName = 'id'
+	sortKeyName2 = 'tag'
+
+	protected listTableFieldDefinitions = ['id', 'version', 'status']
+	protected listTableFieldDefinitions2 = ['tag']
+
+	protected buildTableOutput = buildTableOutput
+	protected translateToId = translateToId
+
+	private getCustomByNamespace = getCustomByNamespace
+
+	async run(): Promise<void> {
+		const { args, argv, flags } = this.parse(CapabilityTranslationsCommand)
+		await super.setup(args, argv, flags)
+
+		if (this.flags.verbose) {
+			this.listTableFieldDefinitions.splice(3, 0, 'locales')
+		}
+
+		let capabilityIdOrIndex = args.id
+		let tagOrIndex = undefined
+		if (args.id) {
+			if (isNaN(args.id)) {
+				// capability ID is not an index, next param must be the version
+				capabilityIdOrIndex = {id: args.id, version: args.version}
+				tagOrIndex = args.tag
+			} else {
+				// capability ID is an index, then version param, if set, must hold the local
+				tagOrIndex = args.version
+			}
+		}
```

Comment:
If we want to be consistent with other capabilities commands we need to change this a bit. In this code from "capabilities.ts":
```js
		const idOrIndex = args.version
			? { id: args.id, version: args.version }
			: args.id
```
When there is no version specified, that doesn't necessarily mean `args.id` is an index...it could still be an id just without specifying the version. The `translateToId` function just fills in `1` for now but the intent is to use the most recent version when versions are available. So, using `isNaN` makes this method behave differently.

Since the `tag` can't be numeric and `version` can, maybe we can use a combination of the number of arguments and the "numberness" of the _last_ one (if there are two) to determine this. Some psuedo code for what I'm thinking:

* If there are 3 arguments, they are id, version, tag.
* If there is 1 argument, it is assigned to `capabilityIdOrIndex` (and `translateToId` will handle whether it's an index or an id that has to have a default version added).
* If there are two arguments:
   * If the second one is a number, it's `id`, `version`
   * otherwise, it's `id`, `tag`

(Also "locale" is missing an "e" in the second comment.)

Reply:
Hmmm, I did not realize that we were defaulting the version to 1 when the capability ID is specified as a string. If we are going to do that then I think we need to output the version in the default table view of the capability. As it is now you don't see which version is being shown:
```
smartthings capabilities xyz.outputVoltage

Capability: xyz.outputVoltage

Attributes: ...
```

This also means that we have an ambiguity, since the nested tag can also be specified as an index, so 
```
$ smartthings capabilities:translations xyz.outputVoltage 2
```
could either mean version 2 of the capability or the second tag of the default version of the capability. I suppose we could drop indexed support for the second parameter, since in this case at least the tags are very short. But other applications might not be. Or we could leave it but require the version in order to reference the tag by index, i.e.
```
$ smartthings capabilities:translations 3 2                    #second  tag of the third capability/version
$ smartthings capabilities:translations xyz.outputVoltage 2    #version two of the capability
$ smartthings capabilities:translations xyz.outputVoltage es   #Spanish tag of the default capability version
$ smartthings capabilities:translations xyz.outputVoltage 2 3  #third tag of version two of the capability
$ smartthings capabilities:translations xyz.outputVoltage 2 es #Spanish tag of version two of the capability
```

Reply:
Another consideration is that defaulting the capability version when not specified gives us no way to list all the versions of a capability from the CLI, once we do have multiple versions